docs/modules/ROOT/pages/index.adoc

```diff
@@ -40,16 +40,16 @@ xref:advisor.adoc[[.card-title]#Advisor# [.card-body]#pass:q[Learn and implement
 [[networks]]
 == Available networks
 
-All Defender components, where applicable, provide support for the Ethereum *Mainnet*, as well as the *Rinkeby*, *Ropsten*, *Kovan*, and *Goerli* test networks, plus the https://www.xdaichain.com/[*xDai*], https://blockscout.com/poa/sokol[*Sokol*], and https://fuse.io/[*Fuse*] chains. Note that some networks may be unavailable depending on your <<tiers,plan>>.
+All Defender components, where applicable, provide support for the Ethereum *Mainnet*, as well as the *Rinkeby*, *Ropsten*, *Kovan*, and *Goerli* test networks, plus the https://www.xdaichain.com/[*xDai*], https://fuse.io/[*Fuse*], https://docs.binance.org/smart-chain/guides/bsc-intro.html[Binance Smart Chain], https://fantom.foundation/what-is-fantom-opera/[Fantom], and https://docs.moonbeam.network/networks/testnet/[Moonbase] chains, along with their respective testnets where applicable. Note that some networks may be unavailable depending on your <<tiers,plan>>.
 
 If there is any other network or layer-2 solution you would like to use from Defender, please <<feedback,reach out to us>>!
 
 [[tiers]]
-== Tiers and quotas
+== Plans and quotas
 
 The Defender free plan is restricted for testnet access only. In addition, you may only create up to five Relayers and Autotasks, import up to five Contracts into Admin, and have up to five Users in your team. The total number of transactions sent from Relayers cannot exceed 30 per hour, and the same limit applies for Autotask runs.
 
-If you want access to Mainnet, xDai, or Fuse, or to increase any of the quota limits from the free plan, please apply via https://openzeppelin.com/apply/[this form] and we'll reach out to you.
+If you want access to Mainnet, xDai, BSC, Fantom, or Fuse, or to increase any of the quota limits from the free plan, you can upgrade your plan to Team or Individual in the Account Plan menu in the top-right menu in Defender.
```

Comment:
Should we mention that there is a free Individual plan?

Reply:
We're mentioning the "Defender free plan", I think it's good enough